config/jobs/kubernetes-sigs/sig-windows/release-1.27-windows.yaml

```diff
@@ -18,11 +18,16 @@ periodics:
   decoration_config:
     timeout: 4h0m0s
   extra_refs:
+  - org: kubernetes-sigs
+    repo: windows-testing
+    base_ref: master
+    path_alias: sigs.k8s.io/windows-testing
+    workdir: true
   - org: kubernetes-sigs
     repo: cluster-api-provider-azure
     base_ref: release-1.7
     path_alias: sigs.k8s.io/cluster-api-provider-azure
-    workdir: true
+    workdir: false
 # Needed when this job switches to using cluser-api-provider@release-1.8
```

Comment:
I have another PR staged that will update the CAPZ branch to release-1.7 but am waiting for 1.27 to release first to minimize changes.